### PR TITLE
Added MicroSIP v3.20.7

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

File: manifests/m/MicroSIP/MicroSIP/3.20.7/MicroSIP.MicroSIP.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://www.microsip.org/download/MicroSIP-3.20.7.exe
+  InstallerSha256: 30CFDC2A9BF193931A4B168104440A31F61CFB13869085E3CDFEBF3F908FBAD4
+  InstallerSwitches:
+    Silent: /S
```
Comment (ItzLevvie) [inline manifests/m/MicroSIP/MicroSIP/3.20.7/MicroSIP.MicroSIP.installer.yaml:21]:
```suggestion
```
I believe these aren't necessary as the client automatically adds the switches based on `InstallerType`.

File: manifests/m/MicroSIP/MicroSIP/3.20.7/MicroSIP.MicroSIP.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: MicroSIP.MicroSIP
+PackageVersion: 3.20.7
+PackageLocale: en-US
+Publisher: MicroSIP
```
Comment (ItzLevvie) [inline manifests/m/MicroSIP/MicroSIP/3.20.7/MicroSIP.MicroSIP.locale.en-US.yaml:7]:
```suggestion
Publisher: www.microsip.org
```

File: manifests/m/MicroSIP/MicroSIP/3.20.7/MicroSIP.MicroSIP.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- sip
+- tel
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
```
Comment (ItzLevvie) [inline manifests/m/MicroSIP/MicroSIP/3.20.7/MicroSIP.MicroSIP.installer.yaml:17]:
```suggestion
  Scope: user
```

File: manifests/m/MicroSIP/MicroSIP/3.20.7/MicroSIP.MicroSIP.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+PublisherUrl: https://www.microsip.org/
+PublisherSupportUrl: https://www.microsip.org/help
+PrivacyUrl: https://www.microsip.org/privacy-policy
+#Author: 
+PackageName: MicroSIP
+#PackageUrl:
```
Comment (ItzLevvie) [inline manifests/m/MicroSIP/MicroSIP/3.20.7/MicroSIP.MicroSIP.locale.en-US.yaml:13]:
```suggestion
PackageUrl: https://www.microsip.org/
```

File: manifests/m/MicroSIP/MicroSIP/3.20.7/MicroSIP.MicroSIP.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+PackageLocale: en-US
+Publisher: MicroSIP
+PublisherUrl: https://www.microsip.org/
+PublisherSupportUrl: https://www.microsip.org/help
+PrivacyUrl: https://www.microsip.org/privacy-policy
+#Author: 
+PackageName: MicroSIP
+#PackageUrl: 
+License: GPLv2
+#LicenseUrl:
```
Comment (ItzLevvie) [inline manifests/m/MicroSIP/MicroSIP/3.20.7/MicroSIP.MicroSIP.locale.en-US.yaml:15]:
```suggestion
LicenseUrl: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by lphimath** [comment]
@ItzLevvie thanks for your comments and helping me out, this is my first contribution so I was especially unsure regarding the silent installer switch. I have applied all of your suggestions.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**